tests/test_idpi/test_grib_decoder.py

```diff
@@ -0,0 +1,24 @@
+# Third-party
+import xarray as xr
+
+# First-party
+from idpi import grib_decoder
+
+
+def test_save(data_dir, tmp_path):
```

Comment:
Wouldn't we need to add tests for all types of supported metadata?

Reply:
Yes, indeed, the coverage of this test is pretty minimal. I wanted to get some feedback on the general approach before starting to write a battery of tests.